docs/operator/did_meta.md

```diff
@@ -53,7 +53,7 @@ order each configured metadata plugin if it supports this KEY.
 
 ## How to develop a custom metadata solution
 
-The module you develop needs to extend the [DidMetaPlugin](/) Abstract
+The module you develop needs to extend the [DidMetaPlugin](https://github.com/rucio/rucio/blob/master/lib/rucio/core/did_meta_plugins/did_meta_plugin_interface.py) Abstract
 class. The methods needed are:
```

Comment:
```suggestion
The module you develop needs to extend the [DidMetaPlugin](https://github.com/rucio/rucio/blob/master/lib/rucio/core/did_meta_plugins/did_meta_plugin_interface.py) 
Abstract class. The methods needed are:
```

This might fix the build issue

Reply:
This version is included now but it looks a few re-runs to get it right, including marking it as a draft an un-drafting it. Might be worth investigating if the linting steps correctly mark things as failures to avoid confusion in the future.